### PR TITLE
DATA SCIENCE | Exploring the Pedestrian Traffic Data and Developmental Activity Footprint to Map and Predict Graffiti Vandalism and Waste Dumping Hotspots across the City of Melbourne |  30% Complete

### PR DESCRIPTION
**Pull request by Supratim-S220205268**
Week 3 Pull Request for Sprint 1

Use Case Includes:
- Use Case Topic
- Scenario for this Use Case
- Learning from this Report
- Introduction and Background
- Importing Data using API V2.1 from City of Melbourne Open Data Library
- Dataset Information such as data types
- Summary Statistics of Datasets
- Data Preprocessing - Preparation and Cleaning
- Exploratory Data Analysis using Visualizations

### CODE REVIEW

**Review by taehwan-jung — APPROVED** [review]
Hi @Supratim-S220205268 ,
Great job! Everything looks good.
I think you’re good to merge it once you get the second approval.

**Review by ananyaKrishnan-2023 — APPROVED** [review]
Good work, please remove the template from top and place it in description along with appropriate elaboration.

Note: Kindly, keep up with this work.

**Pull request merged**
